src/controllers/institution-controller.js

```diff
@@ -0,0 +1,149 @@
+import * as institutionService from '../services/institution-service.js'
+import { models } from '../models/init-models.js'
+import { Sequelize } from 'sequelize'
+import sequelizeConn from '../util/db.js'
+
+export async function fetchProjectInstitutions(req, res) {
+  try {
+    const institutions = await institutionService.fetchInstitutions(
+      req.params.projectId
+    )
+    res.status(200).json({ institutions })
+  } catch (e) {
+    console.error('Problem while getting project institutions (controller).', e)
+    res
+      .status(500)
+      .json({ message: 'Error while getting project institutions.' })
+  }
+}
+export async function addInstitutionToProject(req, res) {
+  const projectId = req.params.projectId
+  const name = req.body.name
+
+  if (name == null || name.replace(/\s/g, '').length == 0) {
+    res.status(404).json({ message: 'Institution cannot be found' })
+    return
+  }
+
+  let institution = await models.Institution.findOne({
+    where: { name: name },
+  })
+
+  try {
+    const transaction = await sequelizeConn.transaction()
+
+    if (institution == null) {
+      institution = models.Institution.build({
+        project_id: projectId,
```

Comment:
There is no `project_id` column on the `institution` table. Can you remove that assignment?

Also there are `user_id` and `active` fields which should be set.